content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -273,6 +273,12 @@ of individual policies are not defined here.
 - [Baseline](https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/policy/baseline-psp.yaml)
 - [Restricted](https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/policy/restricted-psp.yaml)
 
+### Instantiation through Kyverno
+
+- [Overview](https://github.com/kyverno/policies/tree/main/pod-security)
+- [Baseline](https://github.com/kyverno/policies/tree/main/pod-security/default)
+- [Restricted](https://github.com/kyverno/policies/tree/main/pod-security/restricted)
+
```

Comment:
Hi @chipzoller 

I'm concerned that this change might not be in line with the [content guide](https://kubernetes.io/docs/contribute/style/content-guide/) and our policies on third party software.

Would you consider writing a blog article instead, explaining how the two projects work together?

Reply:
Hi, @sftim. I have created this PR upon suggestion by Jim Bugwadia (maintainer of Kyverno) after having had a discussion with @tallclair in which it was his recommendation to amend this page pointing to the Kyverno repo, which I have now done. If this is not the current recommendation, we'd appreciate guidance on how to best incorporate (or not) this information. A blog post is certainly in the realm of possibilities regardless.

Reply:
> how to best incorporate (or not) this information

- A blog post

That's the best way considering the restrictions in the website content guide. We _could_ also look at revising the content guide (which SIG Architecture have a view on as well); that'd take more time and effort I expect.

Consider linking to https://kyverno.io/docs/ rather than to pages on GitHub.
BTW if you'd like help getting that content published inside https://kyverno.io/docs/ then you can file an issue against Kyverno and mention me in it - I can try to give some pointers.

Reply:
The reason we aren't linking to the docs is because we're undergoing a process to move samples to their own repo, and this isn't yet linked into the docs. Kyverno's implementation of the PSS policies are not where the docs currently point. So I'm not sure where this leaves us with regard to inclusion in the Kubernetes docs.

Reply:
Hi @chipzoller .
I agree that adding links to 3rd party products may not be in line with the site's content guide.
Writing a blog article is a great suggestion!
Also, I am not clear on the concept of "instantiation" in this context. 
Is `kyverno` an implementation of a common interface or specification?
Does `kyverno` provide CRDs to declaratively manage Kubernetes resources through policies in a cluster?

Reply:
Hi @kbhawkey . Again, the reason I opened this PR was specifically because @tallclair advised Jim Bugwadia (of Kyverno) to do so. I am really just following up on the best info I have to date.

The concept of "instantiation" is directly from the docs and not a concept/term I coined. An instantiation is a representation of those Pod Security Standards via some means of software, in this case, [Kyverno](https://kyverno.io). OPA/Gatekeeper may have their own but at last glance they did not.

Reply:
Hi @chipzoller. I am still not clear whether this content fits with the Content guide. Some additional feedback:
- I am assuming that this section was created to grow a list of security policy implementations (`As mechanisms mature, they will be defined below on a per-policy basis.`)? The section could use a list to contain the implementations or "instantiations" if there is more than one item to list.
  @tallclair , is this correct?
- What if you provided the links to the policy files similar to the existing example (single file per URL)?
Or are the `Kyverno` files separated into several YAML files?
- The existing example is not using a heading. I don't think the proposed heading seems right.
- The different implementation YAML files could be presented under the project name, for example:
   [**Kyverno**](...)
- [Privileged](somecode1.yaml)
- [Baseline](somecode2.yaml)
- [Restricted](somecode3.yaml) 

```
[**PodSecurityPolicy**](/docs/concepts/policy/pod-security-policy/)
- [Privileged](https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/policy/privileged-psp.yaml)
- [Baseline](https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/policy/baseline-psp.yaml)
- [Restricted](https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/policy/restricted-psp.yaml)
```